reference_data/management/commands/update_all_reference_data.py

```diff
@@ -0,0 +1,40 @@
+import logging
+import os
+from django.core.management.base import BaseCommand
+
+from reference_data.management.commands.update_human_phenotype_ontology import update_hpo
+from reference_data.management.commands.update_dbnsfp_gene import update_dbnsfp_gene
+from reference_data.management.commands.update_gencode import update_gencode
+from reference_data.management.commands.update_gene_constraint import update_gene_constraint
+from reference_data.management.commands.update_omim import update_omim
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = "Loads all reference data"
+
+    def add_arguments(self, parser):
+        parser.add_argument('--omim-key', help="OMIM key provided with registration at http://data.omim.org/downloads", required=True)
```

Comment:
this shouldn't be required of you are skipping omim

Reply:
👍